src/MICore/LaunchOptions.cs

```diff
@@ -377,13 +377,14 @@ public sealed class LocalLaunchOptions : LaunchOptions
 
         private const int DefaultLaunchTimeout = 10 * 1000; // 10 seconds
 
-        public LocalLaunchOptions(string MIDebuggerPath, string MIDebuggerServerAddress, IList<EnvironmentEntry> environmentEntries)
+        public LocalLaunchOptions(string MIDebuggerPath, string MIDebuggerServerAddress, string MIDebuggerArgs, IList<EnvironmentEntry> environmentEntries)
```

Comment:
>public LocalLaunchOptions(string MIDebuggerPath, string MIDebuggerServerAddress, string MIDebuggerArgs, IList<EnvironmentEntry> environmentEntries) [](start = 8, length = 147)

To be on the safe side, I might suggest making this a new constructor, and leave the old one in place (chaining to the new one). There are some parts of the LaunchOptions classes that are used outside of the MIEngine repo, and this may well be one of those parts. This would also let you back out the change to the Android launcher...